src/libasr/asr_verify.cpp

```diff
@@ -420,6 +420,11 @@ class VerifyVisitor : public BaseWalkVisitor<VerifyVisitor>
     }
 
     void visit_Function(const Function_t &x) {
+        ASR::FunctionType_t* x_func_type = ASR::down_cast<ASR::FunctionType_t>(x.m_function_signature);
+        if (x_func_type->m_abi == abiType::Interactive) {
```

Comment:
```suggestion
        if (ASRUtils::get_FunctionType(&x)->m_abi == abiType::Interactive) {
```

Just check if the above works. We can use it if it works.